sycl/plugins/esimd_cpu/pi_esimd_cpu.cpp

```diff
@@ -593,367 +588,525 @@ pi_result piContextCreate(const pi_context_properties *Properties,
                                             const void *PrivateInfo, size_t CB,
                                             void *UserData),
                           void *UserData, pi_context *RetContext) {
-  (void)Properties;
-  (void)NumDevices;
-  (void)Devices;
-  (void)PFnNotify;
-  (void)UserData;
-  (void)RetContext;
-  DIE_NO_IMPLEMENTATION;
+  if (NumDevices != 1) {
+    return PI_INVALID_VALUE;
+  }
+  if (Devices == nullptr) {
+    return PI_INVALID_DEVICE;
+  }
+  if (RetContext == nullptr) {
+    return PI_INVALID_VALUE;
+  }
+
+  cm_support::CmDevice *device = nullptr;
```

Comment:
```suggestion
  cm_support::CmDevice *Device = nullptr;
```
Please, align variable names in the patch.